php-classes/ReCaptcha.php

```diff
@@ -0,0 +1,2246 @@
+<?php
+    
+/*
+Google reCAPTCHA class for PHP
+Author: Tribulant Software
+Author URL: http://tribulant.com
+Version: 1.0	
+*/
+
+/*
+ * http.php
+ *
+ * @(#) $Header: /opt2/ena/metal/http/http.php,v 1.92 2014/08/14 23:17:34 mlemos Exp $
+ *
+ */
+
+define('HTTP_CLIENT_ERROR_UNSPECIFIED_ERROR',       -1);
+define('HTTP_CLIENT_ERROR_NO_ERROR',                 0);
+define('HTTP_CLIENT_ERROR_INVALID_SERVER_ADDRESS',   1);
+define('HTTP_CLIENT_ERROR_CANNOT_CONNECT',           2);
+define('HTTP_CLIENT_ERROR_COMMUNICATION_FAILURE',    3);
+define('HTTP_CLIENT_ERROR_CANNOT_ACCESS_LOCAL_FILE', 4);
+define('HTTP_CLIENT_ERROR_PROTOCOL_FAILURE',         5);
+define('HTTP_CLIENT_ERROR_INVALID_PARAMETERS',       6);
+
+class http_class

```

Comment:
There are a ton of libraries here for recaptcha, it might be easier just to pick another one that is PSR-0 or PSR-4 compliant already: https://packagist.org/search/?q=recaptcha

This one looks clean and like the most popular: https://packagist.org/packages/recaptcha/php5

Modern PHP class distribution practices defined in PSR-0 and its extension PSR-4 dictate one-class-per-file with no side effects like global constants